src/ReactiveUI.SourceGenerator.Tests/ReactiveUI.SourceGenerator.Tests.csproj

```diff
@@ -21,11 +21,12 @@
     <PackageReference Include="Basic.Reference.Assemblies.Net80" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" PrivateAssets="all" VersionOverride="4.11.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" PrivateAssets="all" VersionOverride="4.11.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis" PrivateAssets="all" VersionOverride="4.11.0" />
     <PackageReference Include="FluentAssertions" />
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\ReactiveUI.SourceGenerators\ReactiveUI.SourceGenerators.csproj" />
+    <ProjectReference Include="..\ReactiveUI.SourceGenerators\ReactiveUI.SourceGenerators.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="true" />
```

Comment:
Since we aren't using the actual generator in this case, don't think we need to OutputItemType, it gets created and invoked by the test.

Reply:
Updated, I added it as it was failing to build, but later found it was something else